Enterprise-CMCS/macpro-mdct-hcbs
# CMDCT-4065: Read Only View

### Description
We want to block users from being able to edit fields in the QM reports. The work here was to hide the Create and submit report buttons and make all the editable fields greyed out so the user can read them but not edit. 

### Related ticket(s)
CMDCT-4065

---
### How to test
https://dbls5wcxpk535.cloudfront.net/
1. Log in as any state/admin user that can create reports
2. Walk through the steps to create a report. Just create it and fill out the fields, don't submit
3. Log out and then log back in as any readonly user (helpdeskuser@test.com is one)
5. On the list of reports, click "View" on the one you just created as the state user
6. Click through the report and make sure all the text/date/radio fields are visible, but greyed out and unable to be edited
7. On the Review & Submit tab, make sure that you do not see the "Submit QMS Report" button
